### PR TITLE
OC-8052 Oops Error after REST call to subject data after SOAP data import

### PR DESCRIPTION
**Pull request by jkeremian**
Oops Error after REST call to subject data after SOAP data import

### CODE REVIEW

File: ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java [hunk 1]
```diff
@@ -118,16 +120,16 @@ public ImportCRFDataService(DataSource ds, Locale locale) {
                 for (FormDataBean formDataBean : formDataBeans) {
 
                     CRFVersionDAO crfVersionDAO = new CRFVersionDAO(ds);
```
Comment (yogi-oc) [inline ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java]: Can we remove this if not needed?

File: ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java [hunk 2]
```diff
@@ -118,16 +120,16 @@ public ImportCRFDataService(DataSource ds, Locale locale) {
                 for (FormDataBean formDataBean : formDataBeans) {
 
                     CRFVersionDAO crfVersionDAO = new CRFVersionDAO(ds);
+                	FormLayoutDAO fldao= new FormLayoutDAO<>(ds);
```
Comment (yogi-oc) [inline ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java]: Do we need to create the DAO object in the loop or we can reuse it by creating it only once?

File: ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java (hunk 1 above)
Comment (jkeremian) [inline ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java]: It is need and used on line 146

File: ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java (hunk 2 above)
Comment (jkeremian) [inline ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java]: we can take bot DAOs out of the loop. I can do that.

**Comment by yogi-oc** [comment]
:+1:

**Pull request merged**